test/suites/scheduling/suite_test.go

```diff
@@ -200,7 +200,7 @@ var _ = Describe("Scheduling", Ordered, ContinueOnFailure, func() {
 			env.ExpectCreatedNodeCount("==", 1)
 		})
 		// note: this test can fail on subscription that don't have quota for GPU SKUs
-		It("should support well-known labels for a gpu (nvidia)", func() {
+		It("should support well-known labels for a gpu (nvidia)", Label("gpu"), func() {
```

Comment:
For filtering by label to work you will also need to conditionally adjust the list of selectors in BeforeAll